views/scored_word.erb

```diff
@@ -0,0 +1,10 @@
+<tr>
+  <td><%= @word %></td>
+  <td><%= @score %></td>
+  <td><% if @word %>
+        <% @word.split("").each do |letter| %>
+          <%= "#{letter}: #{Scrabble.score_letter(letter)}" %>
```

Comment:
I would prefer if this logic was not done in the view. At most - the view should be iterating over an array of scores and displaying them, but it should not be calling your class method directly.